geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientReconnectDUnitTest.java

```diff
@@ -387,166 +383,151 @@ public void testDurableReconnectWithBothNonRedundantServersDown() throws Excepti
   public void testBug39332() {
     // create client cache and Send clientReady message
     createCacheClient(2, 20);
-    HashSet redundantServers = new HashSet(pool.getRedundantNames());
+    Set<String> redundantServers = new HashSet<>(pool.getRedundantNames());
     redundantServers.add(pool.getPrimaryName());
 
-    instance.determineAndVerfiyRedundantServers(redundantServers);
-    instance.determineAndVerfiyNonRedundantServers(redundantServers);
+    instance.determineAndVerifyRedundantServers(redundantServers);
+    instance.determineAndVerifyNonRedundantServers(redundantServers);
 
-    LogWriterUtils.getLogWriter()
+    LogService.getLogger()
```

Comment:
`LogService.getLogger()` creates a new instance of `FastLogger` to wrap a Log4J Logger, so you should either 1) convert these statements to `System.out.println` statements, 2) delete them (they don't look very useful), 3) create a `private static final Logger logger = LogService.getLogger()` and then use that. In general I just do 2. delete them.